services/tenant-ui/frontend/package.json

```diff
@@ -15,6 +15,7 @@
     "@vuelidate/validators": "^2.0.1",
     "axios": "^1.3.4",
     "date-fns": "^2.29.3",
+    "json-editor-vue": "^0.10.5",
```

Comment:
Simple vue wrapper for the existing `vanilla-jsoneditor` package we use that saves having to init it in script setup and lets us use a vue component easily